linux-anvil-comp7/Dockerfile

```diff
@@ -31,17 +31,17 @@ RUN /opt/docker/bin/run_commands
 RUN source /opt/conda/etc/profile.d/conda.sh && \
     conda activate && \
     conda create -n test --yes --quiet --download-only \
-        conda-forge::binutils_impl_linux-64 \
-        conda-forge::binutils_linux-64 \
-        conda-forge::gcc_impl_linux-64 \
-        conda-forge::gcc_linux-64 \
-        conda-forge::gfortran_impl_linux-64 \
-        conda-forge::gfortran_linux-64 \
-        conda-forge::gxx_impl_linux-64 \
-        conda-forge::gxx_linux-64 \
-        conda-forge::libgcc-ng \
+        defaults::binutils_impl_linux-64 \
+        defaults::binutils_linux-64 \
+        defaults::gcc_impl_linux-64 \
+        defaults::gcc_linux-64 \
```

Comment:
This is still coming from conda-forge. We should keep this in conda-forge for a while until https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/206 is propagated to feedstocks.

Reply:
> until conda-forge/conda-forge-pinning-feedstock#206 is propagated to feedstocks

I forgot about https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/206! Indeed we should merge this only after that is done.

BTW, are we tracking that as a migration somewhere?